sdk-springboot/src/main/java/io/dapr/springboot/DaprAutoConfiguration.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ */
+
+package io.dapr.springboot;
+
+import org.springframework.boot.autoconfigure.condition.ConditionalOnWebApplication;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+
+/**
+ * Dapr's Spring Boot AutoConfiguration.
+ */
+@Configuration
+@ConditionalOnWebApplication
+@ComponentScan("io.dapr.springboot")
+public class DaprAutoConfiguration {
+}
```

Comment:
You can use `@ComponentScan` here, also, you can replace it with  `@Bean` method and `@ConditionalOnProperty` to provide finer grained control of beans, e.g. you can use property like `dapr.controller.enabled` to control whether to create the controller bean.

Reply:
Would this be a breaking change? If not, you could send a PR after this. You have more knowledge about spring boot than me.

Reply:
No, we can release this version this time, and make additional improments in the future.

Reply:
Thanks. That will also give you time to prepare a PR with your ideas. This one was great BTW. Thanks a lot.